bsp/nuvoton/numaker-m032ki/applications/mnt.c

```diff
@@ -13,12 +13,12 @@
 #include <rtthread.h>
 
 #if defined(RT_USING_DFS)
```

Comment:
```suggestion
#ifdef  DFS_USING_POSIX
```

在4.1.0版本中，unix like的文件系统api是由DFS_USING_POSIX来开启的

Reply:
请同步更改Kconfig中的select依赖项, 同时选定如下两个宏定义
select RT_USING_DFS
select DFS_USING_POSIX
以及在.c文件中的宏判断：
#ifdef DFS_USING_POSIX

Reply:
Done. Please help to revise. thanks.